lib/game_team.rb

```diff
@@ -0,0 +1,36 @@
+class GameTeam
+  attr_reader :game_id,
+              :team_id,
+              :hoa,
+              :result,
```

Comment:
this class is helping to create data from game_teams csv file. This is the singular GameTeam row